server/routes/eventAPI.js

```diff
@@ -1,33 +1,31 @@
 const express = require("express");
 const db = require("../db/models");
+const auth = require("../middleware/auth");
 
 const router = express.Router();
 
 // CREATE meeting
-router.post("/api/event", (req, res) => {
-  // sanitize req data
-  if (req.body.user_id && req.body.duration) {
-    db.EventType.create(req.body)
-      .then((response) => res.send(response))
-      .catch((error) => {
-        console.log(error);
-        res.status(500).send(error);
-      });
-  } else if (!req.body.user_id) {
-    res.status(400).send("User ID is required");
-  } else if (!req.body.duration) {
-    res.status(400).send("Duration is required");
-  }
+router.post("/api/event", auth, (req, res) => {
+    if (req.body.duration) {
+        db.EventType.create({ ...req.body, user_id: req.userId })
```

Comment:
user_id should be camelcase not snakecase, looking into the commits I think this is from a PR that Sina and I didn't review